backend/dal/Services/SecurityDepositReturnService.cs

```diff
@@ -35,6 +35,12 @@ public PimsLease UpdateLeaseDepositReturn(long leaseId, long leaseRowVersion, Pi
         {
             _user.ThrowIfNotAuthorized(Permissions.LeaseEdit);
             ValidateServiceCall(leaseId, leaseRowVersion);
+            var currentHolder = _securityDepositReturnRepository.GetById(deposit.SecurityDepositReturnId).PimsSecurityDepositReturnHolder;
+            if (currentHolder != null)
```

Comment:
Is this possible? or would a keyNotFoundException happen here?

Reply:
If that happens then the update request would be bad since is using the deposit return id to get deposit returns